src/field/email.js

```diff
@@ -15,8 +15,8 @@ export function isEmail(str) {
   return !!result && result[0] !== null;
 }
 
-export function setEmail(m, str) {
-  return setField(m, 'email', str, str => {
+export function setEmail(m, str = '') {
```

Comment:
you're adding a default value here (if str is not present). Isn't this breaking further validation?

Reply:
I just wanted to be sure you would not get null/undefined, but the code is already not throwing [here](https://github.com/auth0/lock/blob/bb92858006eb1b39e38a9fb9000704554dc69997/src/field/email.js#L14), so we're fine.